tests/SettingAppTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+
+namespace Tests;
+
+use DT\Home\Sources\SettingsApps;
+use function DT\Home\container;
+
+class SettingAppTest extends TestCase
+{
+
+    /**
+     * @test
+     */
+    public function it_raw()
+    {
+        $user_apps_services = container()->get( SettingsApps::class );
+        $apps = [ app_factory(), app_factory() ];
+
+        $save = $user_apps_services->save( $apps );
+        $setting_apps = container()->get( SettingsApps::class )->raw();
+
+        $this->assertCount( 2, $setting_apps );
```

Comment:
@Perumal605 , please loop though each $app and make sure it's slug exists in the database. You can use `->find()` for this, or search the $settings_apps array.